### DISCUSSION TITLE
Copy Arc Max

### DISCUSSION THREAD

**Discussion by 12th-devs** [wysh3/zen-premium#2] [Ideas]
I do not have the js knowledge to do this but I think that 5second previews and tidy downloads would be awesome features to have on Zen, if you have the time and desire, there's a project for you :)

https://arc.net/max

**Comment by wysh3** [comment]
hey, thanks for the suggestion!
Im a bit busy right now, but once things settle down, I'll definitely work more on Zen and its JS

**Comment by 12th-devs** [comment]
@wysh3, here is a very buggy tab locker like sigmaOS that ChatGPT made for me

https://github.com/Anoms12/firefox-tab-locker
